shuup/front/apps/personal_order_history/templates/shuup/personal_order_history/macros/order_detail.jinja

```diff
@@ -111,3 +111,11 @@
         {%- if not loop.last %}, {% endif -%}
     {%- endfor -%}
 {%- endmacro -%}
+
+{% macro render_action_buttons(order) %}
+    <div class="pull-right inline">
+        <a class="btn btn-success" href="{{ url("shuup:reorder-order", pk=order.pk) }}">
+            <i class="fa fa-plus"></i> {% trans %}Add all products to cart{% endtrans %}
+        </a>
+    </div>
+{% endmacro %}
```

Comment:
Maybe this macro should be in its own file to allow themes overriding just it. Also if theme has overridden this file (macros/order_detail.jinja) for any Shuup version before this PR, it won't have this new macro. It's possible that the calling side (personal_order_history/order_detail.jinja) is not overridden though and then this missing macro would be a problem. Therefore it would be safer to use a separate file.